pkg/cmd/roachtest/operation/operation_interface.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package operation
+
+import "github.com/cockroachdb/cockroach/pkg/roachprod/logger"
+
+type Operation interface {
+	// ClusterCockroach returns the path to the Cockroach binary on the target
+	// cluster.
+	ClusterCockroach() string
+	Name() string
```

Comment:
When will this be used?

Reply:
If we don't have a spec for any reason, being able to log / report on the Name becomes easy. My version of drt-run used this directly but that's before we moved towards spinning off subprocesses. I think it's worthwhile keeping this in but getting rid of Spec.